frontend/src/app/globals.css

```
@@ -73,7 +73,7 @@
   --color-sidebar-ring: var(--sidebar-ring);
 
   /* Typography - Pilot Space v4 fonts */
-  --font-display: 'Fraunces', Georgia, serif;
+  --font-display: var(--font-fraunces), 'Fraunces', Georgia, serif;
```

Stylelint (17.4.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)